pkg/miniogw/gateway_test.go

```diff
@@ -649,15 +650,15 @@ func runTest(t *testing.T, test func(context.Context, minio.ObjectLayer, storj.M
 
 	planet.Start(ctx)
 
-	layer, m, strms, err := initEnv(ctx, planet)
+	layer, m, strms, err := initEnv(ctx, planet, t)
 	if !assert.NoError(t, err) {
 		return
 	}
 
 	test(ctx, layer, m, strms)
 }
 
-func initEnv(ctx context.Context, planet *testplanet.Planet) (minio.ObjectLayer, storj.Metainfo, streams.Store, error) {
+func initEnv(ctx context.Context, planet *testplanet.Planet, t *testing.T) (minio.ObjectLayer, storj.Metainfo, streams.Store, error) {
```

Comment:
Make `t` the first argument rather than last, usually `testing.T` is the first argument.

Reply:
should it come before `ctx`? I will put it there unless I hear otherwise

Reply:
did this for the above, but can't for this one because the linter wants ctx as the first param

Reply:
actually, the linter complained so I did `ctx, t, planet`

Reply:
me too lol